plugins/debug.c

```diff
@@ -1220,8 +1234,8 @@ static void create_reg_map(void)
 		gtk_misc_set_padding((GtkMisc *)bit_no_read_lbl[i], 0, 10);
 
 		gtk_label_set_line_wrap((GtkLabel *)bit_descrip_list[i], TRUE);
-		gtk_misc_set_alignment((GtkMisc *)bit_descrip_list[i], 0.5, 0.5);
-		gtk_misc_set_padding((GtkMisc *)bit_descrip_list[i], 0, 10);
+		gtk_widget_set_halign((GtkWidget*)bit_descrip_list[i], 0.5);
+		gtk_widget_set_valign((GtkWidget*)bit_descrip_list[i], 0.0);
```

Comment:
Should it be 0.5 instead of 0.0?

Reply:
added the replacement for padding, using margin properties instead